### PR TITLE
Add "development networks" documentation page

### PR DESCRIPTION
**Pull request by samajammin**
## Description

Add "development networks" documentation page

## Related Issue
#1582

### CODE REVIEW

**Comment by samajammin** [comment]
Hey @davidmurdoch @alcuadrado @fzeoli - I'm keen to get your thoughts on this draft. See the issue for context on the thought process here. Thanks in advance! 😄

File: src/content/developers/docs/development-networks/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: Development Networks
+description: An overview of development networks and the tools available to help build Ethereum applications.
+lang: en
+sidebar: true
+---
+
+When building an Ethereum application, you'll want to run it on a local network to see how it works before deploying it.
+
+Similar to how you might run a local server on your computer for web development, you can create a local blockchain instance to test your dapp. This allows for much faster iteration than a public testnet.
```
Comment (alcuadrado) [inline src/content/developers/docs/development-networks/index.md:10]: You can also mention here that in these networks you don't need to deal with ETH, as that's something that many people new to Ethereum don't automatically get.

File: src/content/developers/docs/development-networks/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Quickly fire up a personal Ethereum blockchain which you can use to run tests, execute commands, and inspect state while controlling how the chain operates.
+
+Ganache provides both a a desktop application (Ganche UI), as well as command-line tool (`ganache-cli`). It is part of the Truffle suite of tools.
+
+- [Website](https://www.trufflesuite.com/ganache)
+- [GitHub](https://github.com/trufflesuite/ganache)
+- [Documentation](https://www.trufflesuite.com/docs/ganache/overview)
+
+### Buidler EVM
```
Comment (alcuadrado) [inline src/content/developers/docs/development-networks/index.md:42]: This looks good. We are probably ask you/send a pr asking for a few changes soon, as we are rebranding Buidler.

File: src/content/developers/docs/development-networks/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Available tools {#available-projects}
+
+**Note**: Most [development frameworks](/developers/docs/frameworks/) include a built-in development network. We recommend starting with a framework to [set up your local development environment](/developers/local-environment/).
+
+### Ganache
+
+Quickly fire up a personal Ethereum blockchain which you can use to run tests, execute commands, and inspect state while controlling how the chain operates.
+
+Ganache provides both a a desktop application (Ganche UI), as well as command-line tool (`ganache-cli`). It is part of the Truffle suite of tools.
```
Comment (davidmurdoch) [inline src/content/developers/docs/development-networks/index.md:36]:
```suggestion
Ganache provides both a desktop application (Ganache UI), as well as command-line tool (`ganache-cli`). It is part of the Truffle suite of tools.
```

Comment (samajammin) [reply src/content/developers/docs/development-networks/index.md:36]: Thanks for catching! 😅

File: src/content/developers/docs/development-networks/index.md (hunk 2 above)
Comment (samajammin) [reply src/content/developers/docs/development-networks/index.md:42]: Awesome! Would ❤️  any PRs.

**Comment by samajammin** [comment]
@all-contributors please add @alcuadrado for review.

**Comment by samajammin** [comment]
@all-contributors please add @davidmurdoch for review.

**Comment by samajammin** [comment]
@alcuadrado @davidmurdoch thanks for the feedback! I've incorporated both your changes & will now merge. We're always open to additional feedback & changes, so feel free to reach out with your input or a PR. Cheers.

**Pull request merged**